IBX-6804 - VAT rate assignment in product type (#232)
ibexa/documentation-user: docs/pim/create_product_types.md

For every attribute, you can select **Used for product variants**.
Attributes that have this option are used to [automatically generate product variants](work_with_product_variants.md#generate-variants).

6\. Below the **Attributes** section, in the **VAT rates** area, assign VAT rates that apply to all products of this type.

![Assigning VAT rates](vat_rates_product_type.png "Assigning VAT rates")

!!! tip

>     When your website supports a large number of regions, use the search field to find the right region.

Reviewer: ```suggestion
    When your website supports multiple regions, use the search field to find the right one.
```

>     When your website supports a large number of regions, the search field helps you find the right region.

Reviewer 2: ```suggestion
    When your website supports a large number of regions, use the search field to find the right region.
```
